seacatauth/authz/role/service.py

```
@@ -517,30 +632,91 @@ async def set_roles(self, credentials_id: str, roles: list, tenant: str = "*", i
+			Credentials IDs assigned the role
+		"""
+		query_filter = {"r": role_id}
+		collection = self.StorageService.Database[self.CredentialsRolesCollection]
+		return await collection.count(query_filter)
+
+
+	async def list_role_credentials(
+		self,
+		role_id: str | typing.Iterable,
+		page: int = 0,
+		limit: int = None,
```

Ruff (0.14.3)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)